WebDriverAgentLib/FBApplication.m

```diff
@@ -33,10 +33,22 @@ + (instancetype)fb_activeApplication
   [[[FBRunLoopSpinner new]
     timeout:5]
    spinUntilTrue:^BOOL{
-     return [FBXCAXClientProxy.sharedClient activeApplications].count == 1;
+     return [FBXCAXClientProxy.sharedClient activeApplications].count > 0;
    }];
 
-  XCAccessibilityElement *activeApplicationElement = [[FBXCAXClientProxy.sharedClient activeApplications] firstObject];
+  NSArray<XCAccessibilityElement *> *activeApplicationElements = [FBXCAXClientProxy.sharedClient activeApplications];
```

Comment:
It would be handy to have a comment here on why this filtering is needed

Reply:
Added comment